client/src/components/EditProfile/ProfilePhoto/UploadPhoto.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { ChangeEvent, useState } from 'react';
+import { Avatar, Box, IconButton, Typography, Button } from '@material-ui/core';
+import DeleteIcon from '@material-ui/icons/Delete';
+
+import useStyles from './useStyles';
+
+const UploadPhoto: React.FC = (): JSX.Element => {
+  const classes = useStyles();
+
+  const [selectedImage, setSelectedImage] = useState<File>();
+
+  const imageChange = (e: ChangeEvent<HTMLInputElement>) => {
+    if (e.target.files && e.target.files.length > 0) {
+      setSelectedImage(e.target.files[0]);
+    }
+  };
+
+  return (
+    <Box className={classes.root}>
+      <form>
```

Comment:
I'll usually recommend including this form via the `component="form"` prop of the `Box` component in order to facilitate styling. But it essentially comes down to personal preference 👍🏼

Reply:
@ugur-akin If I want to add like onClick or onSubmit event with this implementation, how would I go about it? I will prefer you respond to me through slack, Thanks.

Reply:
Hi @sholabioku, you can provide any html attribute to mui components and they will pass it down to the primitive html element - meaning you can just add the `onClick` or `onSubmit` attribute to the `Box` component itself 👍🏼